.cspell.json

```diff
@@ -0,0 +1,39 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": [
+    "ciphertext",
+    "evenodd",
+    "gitea",
+    "gonic",
+    "Netrc",
+    "prismjs",
+    "rawurl",
+    "Refspec",
+    "securecookie",
+    "sess",
+    "techknowlogick",
+    "vueuse",
+    "windicss",
+    "xorm",
+    "xormigrate",
+    "zerolog",
+    "Msgf"
+  ],
+  "ignorePaths": [
+    "**/node_modules/**/*",
+    "pnpm-lock.yaml",
+    ".cspell.json",
+    "web/src/assets/locales/**/*",
+    "server/store/datastore/migration/testfiles/sqlite.db",
```

Comment:
```suggestion
    "server/store/datastore/migration/**/*",
```

Reply:
Why should the code / other files in migrations not be checked? I am pretty sure this was intentional. CC @lukashass